### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by orschiro** [ken107/read-aloud#8]
Just wanted to let you know that your extension is just amazing. :-)

**Comment by Limit91** [comment]
True! Just amazing!

**Comment by ken107** [comment]
Thank you for the compliments!  I will keep this issue open forever

**Comment by orschiro** [comment]
@ken107 is there a way to donate to your project?

**Comment by pattulus** [comment]
This issue should definitely be kept open… I also feel the urge to utter my gratitude.

# THANK YOU, KEN!

![](https://media.giphy.com/media/acj7QJGgBBeUg/giphy.gif)

**Comment by orschiro** [comment]
Just wondering since your extension is open-source. 

Couldn't it be implemented into Firefox, too? :-)

Firefox already has a native read aloud feature but it is of very poor quality and adding your read aloud system would be such an improvement!

**Comment by ken107** [comment]
Yes Robert, Firefox is in the TODO list.  I'll put up a donate button on the site later.  Thanks!

**Comment by ajhsu** [comment]
Thanks @ken107 , it's really helpful to me to read online articles. Before I found this tool, I was using macOS's built-in TTS and soon I found that Google's Voice is better in details.
And I do really love the text highlight feature by the way!

**Comment by ken107** [comment]
Good to hear you like it!

**Comment by jjsarton** [comment]
Thank a great extension,

**Comment by peterberbec** [comment]
I have been looking for this exact extension for years and I cannot thank you enough. Where is the donate button???
